packages/wrangler/src/d1/list.tsx

```diff
@@ -12,18 +12,26 @@ import type {
 import type { Database } from "./types";
 
 export function Options(d1ListYargs: CommonYargsArgv) {
-	return d1ListYargs.epilogue(d1BetaWarning);
+	return d1ListYargs
+		.option("json", {
+			describe: "return output as clean JSON",
+			type: "boolean",
```

Comment:
For the boolean types, I think it implicitly defaults to false, I have seen the it explicitly set to false as well. Should we have a consistent way to handle this?

Reply:
Happy to be explicit about this